src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -60,10 +60,23 @@ public final class SunEC extends Provider {
 
     private static final long serialVersionUID = -2279741672933606418L;
 
+    // flag indicating whether the full EC implementation is present
+    // (when native library is absent then fewer EC algorithms are available)
+    private static boolean useFullImplementation = true;
+
+    private static final boolean nativeCryptTrace = NativeCrypto.isTraceEnabled();
+
+
```

Comment:
Just one blank line between declarations.

We'll also want this:
```java
    private static final boolean isAIX = "AIX".equals(System.getProperty("os.name"));
```
to avoid repeatedly fetching that property.

Reply:
Fixed.